constellations/orion/src/domains/class.go

```diff
@@ -48,10 +48,10 @@ func (class *Class) Validate() error {
 		return errors.New("invalid times")
 	}
 
-	//Price validation
-	//Both valid
-	if priceLump.Valid && pricePerSession.Valid {
-		return errors.New("invalid price: both valid")
+	// Price validation
+	// Both valid
+	if priceLump.Valid == pricePerSession.Valid {
```

Comment:
I see this was my fault. I originally told Chujia that a class COULD be free or something. But that almost never happens. If we have a "free" class, it's usually like a one-off seminar, which is outside the scope of what a "class" represents. Thanks for fixing this.